### PR TITLE
Update readme to fit webpack v2

### PR DESCRIPTION
**Pull request by colevoss**
## What:
Updates README.md to adhere to webpack v2 loader updates. [webpack/webpack issue #3195](https://github.com/webpack/webpack/issues/3195)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,19 @@ Extracts SourceMaps for source files that as added as `sourceMappingURL` comment
...
 
 ``` javascript
 module.exports = {
   module: {
-    preLoaders: [
+    rules: [
       {
         test: /\.js$/,
-        loader: "source-map-loader"
+        loader: "source-map-loader",
```
Comment (TheLarkInn) [inline README.md]: can this be `use: ["source-map-loader"]`

Comment (TheLarkInn) [inline README.md]: I believe it can also simply be a string as well last time I checked.

Comment (colevoss) [inline README.md]: For sure. I used [this](https://webpack.js.org/guides/migrating/#module-preloaders-and-module-postloaders-was-removed) as the example so you might want to update that as well.

**Comment by colevoss** [comment]
@TheLarkInn checkout that change if you will. Also sorry for the vim keys at the end of the commit message.

**Comment by TheLarkInn** [comment]
Looks good for now. Still has more work to do for webpack/webpack#3195

**Pull request merged**